website/pages/docs/troubleshooting.md

```diff
@@ -44,4 +44,12 @@ You can read more about this [here](/docs/advanced-topics/running-cloudquery-in-
 
 A few specific tables in AWS are quite slow to sync. You can try skipping them if you don't need this data.
 Take a look at the [skip_tables list](/docs/plugins/sources/aws/configuration#skipping-tables-with-configuration-parameters).
-If syncs are still taking a long time, you can also take a look at our [performance tuning guide](/docs/advanced-topics/performance-tuning).
\ No newline at end of file
+If syncs are still taking a long time, you can also take a look at our [performance tuning guide](/docs/advanced-topics/performance-tuning).
+
+### I am running `cloudquery sync` locally, but it is taking a long time / doesn't seem to finish
+
+(If running the AWS plugin, try the `skip_tables` solution from the previous paragraph first).
+
+The large cloudquery plugins, such as AWS, make a lot of DNS queries (for example, the AWS plugin makes a DNS query per `num_regions * num_services`).
+Some weaker DNS endpoints may not be able to handle this load (especially plausible for small "home-router" DNS endpoints).
+You can try to point your machine to a different DNS endpoint (such as the reliable [Google DNS Server](https://developers.google.com/speed/public-dns)).
```

Comment:
```suggestion
As a workaround. you can try pointing your machine to a different DNS server (such as the reliable [Google DNS Server](https://developers.google.com/speed/public-dns)).
```

Reply:
I'm not sure this is a 'workaround' - it's just a solution if your router is bad :)
I fixed all the rest of the comments.